### PR TITLE
T03

### PR DESCRIPTION
**Pull request by Ashish02112001**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+// ------------------------------------------------------------------------
+// Program.cs
+// program to display the Fibonacci series given an input
+// --------------------------------------------------------------------------------------------
+
+
+namespace Training {
+   #region Program ------------------------------------------------------------------------------
+   /// <summary>Fibonaccci Series</summary>
    internal class Program {
```
Comment (tmmvigneshm) [inline Program.cs:13]: Do we need this 'internal' access modifier here ?

**Comment by tmmvigneshm** [comment]
@Ashish02112001 : 
Also please refactor you code to comply with the comments I have mentioned for the previous assignments.

File: Program.cs (hunk 1 above)
Comment (Ashish02112001) [reply Program.cs]: Here it is not needed.

**Comment by tmmvigneshm** [comment]
@Ashish02112001 
Your logic need to be corrected:
Read once again the assignment question, you were asked to print Fibonacci series for  F(n)
F(0) should be 0
F(1) should be 1
F(2) = F(1) + F(0)

Recheck you logic and apply the changes.
Refer https://en.wikipedia.org/wiki/Fibonacci_sequence

**Pull request closed without merging**